### ISSUE TITLE
config command raises error and settings require restart to take effect

### ISSUE THREAD

**Issue by sanzgiri** [117/alpaca-terminal#1]
I noticed that setting any of the config parameters results in the following error:

```
> config credentials.key xxxx
xxxx
TypeError: Cannot read property 'credentials' of undefined
    at prompt.config (/usr/local/lib/node_modules/alpaca-terminal/dist/src/main.js:105:35)
    at /usr/local/lib/node_modules/alpaca-terminal/dist/src/main.js:47:38
    at Interface._onLine (readline.js:335:5)
    at Interface._line (readline.js:666:8)
    at Interface._ttyWrite (readline.js:1010:14)
    at ReadStream.onkeypress (readline.js:213:10)
    at ReadStream.emit (events.js:314:20)
    at emitKeys (internal/readline/utils.js:335:14)
    at emitKeys.next (<anonymous>)
    at ReadStream.onData (readline.js:1144:36)
```
I got this with all parameters, but they are somehow saved because "config" lists them correctly. If you try to immediately get account status, it fails with the following error:

```
access key verification failed : access key not found (Code = 40110000)
```

However, after I exited and restarted `alpaca-terminal`, the commands seem to be working correctly.

**Comment by sanzgiri** [comment]
I am using npm 6.14.8 on a mac and alpaca-terminal 1.6.2

**Comment by 117** [comment]
Fixed in b5bee5f6225530497b6d1123817f631a1b35055a, which is included in version `1.6.3` thank you for reporting this issue. 🙂

**Comment by sanzgiri** [comment]
Thanks for the quick fix!
